site/index.ejs

```diff
@@ -47,6 +47,11 @@ ga('send', 'pageview');
     <div class="js-app">
       <% if (typeof bodyContent !== 'undefined') { %><%= bodyContent %><% } %>
     </div>
+    <% if (typeof state !== 'undefined') { %>
+      <script type="text/javascript">
```

Comment:
I think this is enough to make it safe.

    JSON.stringify(state).replace(/<\/script/g, '<\\/script').replace(/<!--/g, '<\\!--');

Need to make sure that script tags are escaped, https://medium.com/node-security/the-most-common-xss-vulnerability-in-react-js-applications-2bdffbcc1fa0#.n0rfap4jv